src/pages/session/components/customEstimate.tsx

```diff
@@ -2,23 +2,26 @@ import * as React from "react";
 import { Button } from "azure-devops-ui/Button";
 import "./customEstimate.scss"
 
-export const CustomEstimate: React.FC<{
+interface Props {
```

Comment:
```suggestion
interface CustomEstimateProps {
```
Maybe better name than just props ? up to you to decide